pkg/tracing/instrumentor.go

```diff
@@ -0,0 +1,14 @@
+package tracing
+
+import (
+	"net/http"
+
+	"google.golang.org/grpc"
+)
+
+//go:generate mockery --name Instrumentor
+type Instrumentor interface {
```

Comment:
Can we have some comments here what this is and when this is used? Or is this some basically internal interface? Looks like some kind of wrapper or middleware to me